bookface/posts/models/posts.py

```diff
@@ -0,0 +1,13 @@
+from bookface import db
+from datetime import datetime
+from sqlalchemy import ForeignKey
+
+class Post(db.Model):
+    __tablename__ = "posts"
+    post_id = db.Column(db.Integer, primary_key=True)
+    description = db.Column(db.Text)
+    creation_date = db.Column(db.DateTime, default=datetime.now())
```

Comment:
db.Model should automatically provide us with created_at column. So this line is useless